### DISCUSSION TITLE
Daily notes highlight

### DISCUSSION THREAD

**Discussion by cgael** [BryanJenksCommunity/FAQ#181] [Q&A] [answered]
hi,
I’m new on obsidian and just saw the video of Bryan Jenks on his new daily note system and so I wanted to recover and use it in my way but I can not use the highlight model, when I use it with its vault and pluggin , everything works but as soon as I use my vault ca and not in the dataview ...
here are my pluggin: Calendar, Dataview, periodic notes, templater. 
is there a video to see how Bryan Jenks set up this program or a pluggin missing or something to add?

**Comment by tallguyjenks** [comment]
im going to need pictures to figure out what you're talking about, what exactly isn't working?

please provide screenshots and frame your statements in the following manner for clarity:

> "I changed `x` to `y` in order to achieve `z`, and i got this error (attached screenshot of error)"

**Reply by cgael** [reply]
j'aimerais crée ce qu'il y a dans l'obsidian noir, j'ai donc copier les template (daily, weekly , monthly, anualy and highlight), mais comme vous pouvez le voir, quand j'ajoute la template highlight pour marquer un évenement important de ma journée , cela ne fonctionne pas dans ma vault...
voici un screen vidéo et mes pluggings 

https://github.com/BryanJenksCommunity/FAQ/assets/150393661/4e9124a8-7b28-4a17-b04f-fe41b68d6edb

![Capture d’écran 2023-11-10 à 14 38 46](https://github.com/BryanJenksCommunity/FAQ/assets/150393661/d07c92ff-1039-42d3-8d42-b890dc0a203e)

**Reply by tallguyjenks** [reply]
Interesting. if dataview is installed and its the same template being used then there is no reason that the logic of the query shouldn't work and the highlight not appear there. i am wondering if dataview is just taking a while to load the updated value of the highlight and lagging on it (it has done this to me).

Try to add the highlight to the light themed obsidian then close and restart the obsidian application and see if it was just a matter of dataview updating slowly.

if not, send me the entire file from the light themed editor in a reply comment wrapped in a code fence like:

``````
```md
my content here
```
``````

also show me a screen shot of your settings for the dataview plugin

**Reply by cgael** [reply]
J'ai tout fait comme tu m'as dit et rien , pourtant j'ai pris exactement les même donnée, mise en forme et paramètre que Bryan Jenks....
C'est nul... il existe pas une vidéo pour voir comment programmé cela et quel pluggin sont nécessaire ?
J'utilise le thème "things" et je ne comprend pas ce que tu veux que je t'envoie ...
<img width="1010" alt="Capture d’écran 2023-11-10 à 20 24 24" src="https://github.com/BryanJenksCommunity/FAQ/assets/150393661/cee557b5-192e-4a86-ac56-9b0f84267ecc">
<img width="1010" alt="Capture d’écran 2023-11-10 à 20 24 31" src="https://github.com/BryanJenksCommunity/FAQ/assets/150393661/a89ddfeb-ee63-4ce4-aa65-046edcf555f7">
<img width="1010" alt="Capture d’écran 2023-11-10 à 20 24 35" src="https://github.com/BryanJenksCommunity/FAQ/assets/150393661/449a93cb-0d4f-4dde-92c3-56019bfe781e">

**Reply by tallguyjenks** [reply, marked as answer]
i think i see the problem now, in your vault your daily notes folder is not at the root of the vault but inside the ![image](https://github.com/BryanJenksCommunity/FAQ/assets/29872822/f0647103-bae6-42f1-882f-0f330744995d) folder

`Journal` is in another folder:

![image](https://github.com/BryanJenksCommunity/FAQ/assets/29872822/fc0970ff-32d0-4ca4-b8a2-67fe57f24398)

mine is here:

![image](https://github.com/BryanJenksCommunity/FAQ/assets/29872822/0249a9d8-869a-4cec-a982-244ee0dd87fd)

so in the daily note template you need to update this dataview query to include the name of the folder the `Journal` folder is held in.

![image](https://github.com/BryanJenksCommunity/FAQ/assets/29872822/61551d28-d3d5-4ad6-b610-46fa60744073)

to what ever the name of that folder is:

``````
```dataview
TABLE highlights
FROM "Le second cerveau de Gaëlou 🧠️/Journal/Daily"
WHERE this.file.name = file.name
AND highlights != null
```
``````
